vms/registration/views.py

```diff
@@ -200,7 +244,28 @@ def post(self, request):
                             'registered': self.registered,
                             'phone_error': self.phone_error,
                             'organization_list': self.organization_list,
+                            'country_list': country_list,
                         })
         else:
             return render(request, 'home/home.html', {'error': True})
 
+def load_states(request):
+    """
+    View to return states belonging to the selected country on ajax call
+
+    """
+
+    country_id = request.GET.get('country')
+    states = Region.objects.filter(country_id=country_id).order_by('name')
+    return render(request, 'registration/state_dropdown_list_options.html',{'states':states})
+
+def load_cities(request):
+    """
+    View to return cities belonging to the selected country and state on ajax call
+
+    """
+    country_id = request.GET.get('country')
+    state_id = request.GET.get('state')
+    cities = City.objects.filter(country_id=country_id,region_id=state_id).order_by('name')
+    return render(request, 'registration/city_dropdown_list_options.html', {'cities': cities})
```

Comment:
Hey, what if a country doesn't have states? For example, Aruba, it's a small country that has only cities. In this case, we won't be able to select cities as the city's drop-down menu will be disabled.

Reply:
done